test/unittest/statistics/dds/StatisticsQosTests.cpp

```diff
@@ -111,10 +219,10 @@ class StatisticsFromXMLProfileTests : public ::testing::Test
                 static_cast<TestDomainParticipantImpl*>(domain_statistics_participant_impl);
         ASSERT_NE(test_statistics_domain_participant_impl, nullptr);
 
-        // Get PublisherImpl
-        _statistics_publisher_impl =
-                test_statistics_domain_participant_impl->get_publisher_impl();
-        ASSERT_NE(_statistics_publisher_impl, nullptr);
+        // Get Publisher
+        _statistics_publisher =
```

Comment:
I know it is not related to the changes but I think we should unify the code style according to [Fast DDS [guidestyle](https://github.com/eProsima/cpp-style#code-style-for-eprosima-open-source-c-projects). Private class member should have a trailing underscore, but there is no rule to add underscore before the variable name.